doc/02-installation.md

```diff
@@ -450,749 +428,210 @@ yum install nagios-plugins-all
+systemctl restart icingadb-redis
 ```
 
-![Vim with syntax highlighting](images/installation/vim-syntax.png "Vim with Icinga 2 syntax highlighting")
-
-
-### Using Nano <a id="configuration-syntax-highlighting-nano"></a>
-
-Install the package `nano-icinga2` with your distribution's package manager.
-
-<!-- {% if debian or ubuntu or raspbian %} -->
-<!-- {% if not icingaDocs %} -->
-#### Debian / Ubuntu / Raspbian
-<!-- {% endif %} -->
-**Note:** The syntax files are installed with the `icinga2-common` package already.
-<!-- {% endif %} -->
-
-<!-- {% if centos %} -->
-<!-- {% if not icingaDocs %} -->
-#### CentOS
-<!-- {% endif %} -->
+### Enable Icinga DB Feature <a id="enable-icinga-db-feature"></a>
 
-#### CentOS 8
-```bash
-dnf install nano-icinga2
-```
+With the [Icinga DB feature](https://icinga.com/docs/icinga-2/latest/doc/14-features/#icinga-db) enabled,
+Icinga 2 publishes all of its monitoring data to the Redis server. This includes configuration and
+its runtime updates via the Icinga 2 API, check results, state changes, downtimes, acknowledgments, notifications and
+other events such as flapping.
 
-#### CentOS 7
-```bash
-yum install nano-icinga2
-```
-<!-- {% endif %} -->
+![Icinga DB Icinga 2](images/icingadb/icingadb-icinga2.png)
 
-<!-- {% if rhel %} -->
-<!-- {% if not icingaDocs %} -->
-#### RHEL
-<!-- {% endif %} -->
-#### RHEL 8
-```bash
-dnf install nano-icinga2
-```
+Icinga 2 installs the feature configuration file to `/etc/icinga2/features-available/icingadb.conf`,
+pre-configured for a local setup.
+Update this file in case Redis is running on a different host or to set credentials.
+All available settings are explained in the [Icinga DB object](09-object-types.md#objecttype-icingadb) chapter.
 
-#### RHEL 7
-```bash
-yum install nano-icinga2
-```
-<!-- {% endif %} -->
+To enable the `icingadb` feature use the following command:
 
-<!-- {% if fedora %} -->
-<!-- {% if not icingaDocs %} -->
-#### Fedora
-<!-- {% endif %} -->
 ```bash
-dnf install nano-icinga2
+icinga2 feature enable icingadb
 ```
-<!-- {% endif %} -->
 
-<!-- {% if sles or opensuse %} -->
-<!-- {% if not icingaDocs %} -->
-#### SLES / openSUSE
-<!-- {% endif %} -->
-```bash
-zypper install nano-icinga2
-```
-<!-- {% endif %} -->
+Restart Icinga 2 for these changes to take effect:
 
-<!-- {% if amazon_linux %} -->
-<!-- {% if not icingaDocs %} -->
-#### Amazon Linux 2
-<!-- {% endif %} -->
 ```bash
-yum install nano-icinga2
+systemctl restart icinga2
 ```
 <!-- {% endif %} -->
 
-Copy the `/etc/nanorc` sample file to your home directory.
-
-```bash
-cp /etc/nanorc ~/.nanorc
-```
-
-Include the `icinga2.nanorc` file.
-
-```
-$ vim ~/.nanorc
-
-## Icinga 2
-include "/usr/share/nano/icinga2.nanorc"
-```
-
-Test it:
-
-```bash
-nano /etc/icinga2/conf.d/templates.conf
-```
-
-![Nano with syntax highlighting](images/installation/nano-syntax.png "Nano with Icinga 2 syntax highlighting")
+### Install Icinga DB Daemon <a id="install-icinga-db-daemon"></a>
```

Comment:
This section shows up on the Raspbian/Fedora/OpenSUSE pages without the other Icinga DB sections.